lib/ProductOpener/Web.pm

```diff
@@ -41,14 +41,37 @@ use utf8;
 use Exporter qw(import);
 
 use ProductOpener::Display qw/:all/;
+use ProductOpener::Store qw(:all);
```

Comment:
Do we really need all those modules?

One thing that would be very nice to do is to remove the circular dependencies that we currently have (module A that use module B, and module B that uses module A). So we should only use the minimum set of modules that are actually used.

Reply:
I added them because we are gonna need them for the next functions we are gonna move to Web.pm. Also, I was not sure where `$server_options`  is present.

Reply:
$server_options is imported from Config.pm